templates/order/check.html.twig

```diff
@@ -71,6 +73,24 @@
                     <label><input type="radio" name="ppc_qn" value="no"/> No</label>
                 </div>
             </div>
+            <div style="display:none;" id="syncope-qn">
+                <h3>Has the participant ever felt dizzy, nauseous, or lost consciousness when having blood drawn?</h3>
+                <div class="radio">
+                    <label><input type="radio" name="syncope" value="yes"/> Yes</label>
+                </div>
+                <div class="radio">
+                    <label><input type="radio" name="syncope" value="no"/> No</label>
+                </div>
+                <div style="display:none;" id="syncope-sub-qn">
+                    <h3>Would the participant like to continue with a full HPO Biobank order or a modified order with urine and saliva only?</h3>
+                    <div class="radio">
+                        <label><input type="radio" name="syncope_sub" value="yes"/> Full HPO Biobank Order</label>
+                    </div>
+                    <div class="radio">
+                        <label><input type="radio" name="syncope_sub" value="no"/> Modified Order - {% if app.session.get('siteType') != 'dv' %} Saliva and Urine {% endif %} Only</label>
```

Comment:
It looks like we were already previously doing this, but would it make sense to pass in the site type to the view from the controller, instead of accessing the session directly?